### PR TITLE
RAT Test timestamp (EXPOSUREAPP-6895)

### PR DESCRIPTION
**Pull request by mtwalli**
- Rapid antigen test should always display `testedAt` timestamp
- wording for Rapid antigen test should be `Durchgeführt am: _time_`
Screenshot 2021-05-19 at 12 49 35

- Some code refactoring 

Testing:
1- In home screen RAT positive and negative cards should display testedAt time - No text change 
2- Pending test result screen should display `Durchgeführt am: _testedAt_` for RAT test , PCR stay as is

### CODE REVIEW

**Review by AlexanderAlferov — APPROVED** [review]
LGTM 🚀 
Screenshot 2021-05-19 at 16 39 35
Screenshot 2021-05-19 at 16 35 58

**Review by chris-cwa — APPROVED** [review]
👍

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/testresult/pending/SubmissionTestResultPendingFragment.kt [hunk 1]
```diff
@@ -55,28 +55,19 @@ class SubmissionTestResultPendingFragment : Fragment(R.layout.fragment_submissio
...
-                }
-                CoronaTest.Type.RAPID_ANTIGEN -> {
-                    testResultPendingStepsWaitingAntigenResult.isVisible = true
-                    testResultPendingStepsRatAdded.isVisible = true
-                    testResultPendingStepsWaitingPcrResult.isVisible = false
-                    testResultPendingStepsPcrAdded.isVisible = false
-                }
-            }
+            testResultPendingStepsWaitingPcrResult.isVisible = isPcr
+            testResultPendingStepsPcrAdded.isVisible = isPcr
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/testresult/pending/SubmissionTestResultPendingFragment.kt:70]: FYI -> https://sonarcloud.io/component_measures?id=corona-warn-app_cwa-app-android&metric=new_duplicated_lines_density&pullRequest=3192&selected=corona-warn-app_cwa-app-android%3ACorona-Warn-App%2Fsrc%2Fmain%2Fjava%2Fde%2Frki%2Fcoronawarnapp%2Fui%2Fsubmission%2Ftestresult%2Finvalid%2FSubmissionTestResultInvalidFragment.kt&view=list

**Comment by dsarkar** [comment]
Fixes https://github.com/corona-warn-app/cwa-app-android/issues/3214

**Pull request merged**